.travis.yml

```diff
@@ -14,6 +14,7 @@ python:
 install: "make"
 # command to run tests
 script:
+  - make test-readme
```

Comment:
This shouldn't be run on Python 2.6. It seems `readme-renderer` doesn't support 2.6.